spec/spotify/artist_spec.cr

```diff
@@ -38,5 +38,10 @@ describe Spotify::Artist do
       expect(artist.images.size).to eq(4)
       expect(artist.images).to be_a(Array(Spotify::Image))
     end
+
+    it "populates the genres" do
+      expect(artist.genres.size).to eq(2)
+      expect(artist.genres).to be_a(Array(String))
+    end
```

Comment:
@gabrielalmir10 can you just check the content of the array here? just to be sure.

`expect(artist.genres[0]).to eq("...")`

After that I can merge it :dancer: